packages/@icon-magic/config-reader/src/config-loader.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+const importFresh = require('import-fresh');
```

Comment:
Let's be good typescript citizens and consider writing types for this module :) 
Check out the local-types in CSS Blocks: https://github.com/linkedin/css-blocks/tree/master/types-local